Lykke.Service.LykkeService/client/Lykke.Service.LykkeService.Client/LykkeServiceClient.cs

```diff
@@ -4,12 +4,18 @@ namespace Lykke.Service.LykkeService.Client
 {
     public class LykkeServiceClient : ILykkeServiceClient
     {
-        //public IControllerApi Controller { get; }
-        
+        //In case of several api controllers in your app - uncomment and use multiple APIs code below instead of one private field.
+        private ILykkeServiceClient _api;
```

Comment:
According to written above, I think will be better to make multiple APIs example by default. E.g. replace ILykkeServiceClient with IControllerApi1 and IControllerApi2 and expose Api1 & Api2 properties here